index.js

```diff
@@ -3,7 +3,9 @@ module.exports = probotPlugin
 const handlePullRequestChange = require('./lib/handle-pull-request-change')
 
 function probotPlugin (robot) {
-  robot.on('pull_request.opened', handlePullRequestChange.bind(null, robot))
-  robot.on('pull_request.edited', handlePullRequestChange.bind(null, robot))
-  robot.on('pull_request.synchronize', handlePullRequestChange.bind(null, robot))
-}
+ robot.on([
+	'pull_request.opened',
+	'pull_request.edited',
+	'pull_request.synchronize'
+ ], handlePullRequestChange.bind(null, robot))
+}
```

Comment:
Looks great, this will work 👍 

The error that you see in the pull request:
![image](https://user-images.githubusercontent.com/39992/31806030-396adb4a-b51a-11e7-87d8-61328ae14ecf.png)

is due to syntax linting. It requires 2 spaces to be used for indentation instead of tabs, could you fix that?

Reply:
which line is giving build fail?